solutions/CSES Increasing Subsequence II.mdx

```diff
@@ -58,6 +64,77 @@ int main () {
 	cout << sol << '\n';
 }
 ```
+Java implementation without template:
```

Comment:
Use the `<LanguageSection>` and `<JavaSection>` tags (see [step 2](https://usaco.guide/general/contributing?lang=java#adding-a-solution))